tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -38,7 +38,7 @@ final class ClosureLinkerBackend(
     outputMode: OutputMode,
```

Comment:
You can choose es 5 global or isolated. And assuming we can upgrade gcc someday, es 6 will also become an option.